pkg/network/ovn_kubernetes_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"slices"
```

Comment:
I think linter is unhappy because of this package, which I think requires a linter version update to make it work. Since we don't have much time to merge extra PRs, maybe we could replace this package with `"k8s.io/utils/strings/slices"` which is already used in the repo and provides a similar `Contains` function?

Reply:
I checked that the `Contains` function from `"k8s.io/utils/strings/slices"` expects `[]string` but `enabledFeatureGates` is of type  `[]configv1.FeatureGateName`, so it's not directly compatible here. Should we use a `map` instead of a slice? 


```go
enabledFeatureGates      map[configv1.FeatureGateName]bool
```

Reply:
ah sorry for the wrong pointer, I would suggest `"k8s.io/apimachinery/pkg/util/sets"` in that case

Reply:
Okay, I've updated to use `"k8s.io/apimachinery/pkg/util/sets"`